qutebrowser/misc/editor.py

```diff
@@ -128,6 +117,19 @@ def edit(self, text, caret_position=None):
         line, column = self._calc_line_and_column(text, caret_position)
         self._start_editor(line=line, column=column)
 
+    def _on_file_changed(self, path):
+        encoding = config.val.editor.encoding
+        try:
+            with open(path, 'r', encoding=encoding) as f:
+                text = f.read()
+        except OSError as e:
+            # NOTE: Do not replace this with "raise CommandError" as it's
+            # executed async.
+            message.error("Failed to read back edited file: {}".format(e))
+            return
+        log.procs.debug("Read back: {}".format(text))
+        self.editing_finished.emit(text)
```

Comment:
Hmm, I wonder whether we should rename this signal - `editing_finished` sounds like something which can't occur multiple times. Maybe it should be called `file_updated` or so now?